cumulusci/tasks/salesforce/tests/test_ActivateProcessBuilderProcesses.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+from unittest import mock
+import json
+import responses
+from .util import create_task
+from cumulusci.tasks.salesforce.ActivateFlowProcesses import ActivateFlowProcesses
+
+task_options = {
+    "description": "Activates Flows identified by a given list of Developer Names",
+    "developer_names": ["Auto_Populate_Date_And_Name_On_Program_Engagement", "ape"],
+    "required": True,
+}
```

Comment:
description and required belong in the task_options dict on the task class itself which defines what options the task accepts. Not here where you are preparing specific _values_ of those options to use with an instance of the task.